build.boot

```diff
@@ -203,95 +273,47 @@
 
+(defn- task-constructor
+  [task-name func]
+  (let [db (get-wombats-db-name)]
+    (if (can-run-command? task-name)
+      (do
+        (println (format (boot-constants :task-confirm) task-name db))
+        (let [input (read-line)]
+          (if (= input "Yes")
+            (func)
+            (println (format (boot-constants :task-cancel) task-name db)))))
+      (println (format (boot-constants :task-block) task-name db)))))
+
 (deftask refresh-db-functions
   "resets the transactors in the db"
   []
-  (System/setProperty "APP_ENV" "dev")
-  (-> (build-connection-string)
-      (d/connect)
-      (db-fns/seed-database-functions)))
-
-(deftask seed-local
-  "Seeds the dev dynamo db"
-  []
-  (System/setProperty "APP_ENV" "dev")
-
-  (-> (build-connection-string)
-      create-db!
-      seed-db!))
-
-(deftask refresh-local
-  "resets the database"
-  []
-  (System/setProperty "APP_ENV" "dev")
+  (task-constructor "refresh-db-functions"
+                    #(-> (build-connection-string)
+                         (d/connect)
+                         (db-fns/seed-database-functions))))
 
-  (-> (build-connection-string)
-      refresh-db!))
-
-(deftask delete-local
-  "deletes the local db"
+(deftask seed
+  "Seed the current database set through WOMBATS_ENV"
   []
-  (System/setProperty "APP_ENV" "dev")
-
-  (-> (build-connection-string)
-      delete-db!))
+  (task-constructor "seed"
+                    #(-> (build-connection-string)
+                         create-db!
+                         seed-db!)))
 
-(deftask seed-dev
-  "Seeds the dev dynamo db"
+(deftask refresh
+  "Refresh the current database set through WOMBATS_ENV"
   []
-  (System/setProperty "APP_ENV" "dev-ddb")
-
-  (-> (build-connection-string)
-      create-db!
-      seed-db!))
-
-(deftask refresh-dev
-  "Resets the dev dynamo db"
-  []
-  (System/setProperty "APP_ENV" "dev-ddb")
-
-  (-> (build-connection-string)
-      refresh-db!))
-
-(deftask delete-dev
-  []
-  (System/setProperty "APP_ENV" "dev-ddb")
-
-  (-> (build-connection-string)
-      delete-db!))
+  (task-constructor "refresh"
+                    #( -> (build-connection-string)
+                      refresh-db!)))
 
-(deftask seed-qa
-  "Seeds the dev dynamo db"
+(deftask delete
+  "Deletes the current database set through WOMBATS_ENV"
   []
-  (System/setProperty "APP_ENV" "qa-ddb")
-
-  (-> (build-connection-string)
-      create-db!
-      seed-db!))
-
-(deftask refresh-qa-ddb
-  "Resets the qa dynamo db"
-  []
-  (System/setProperty "APP_ENV" "qa-ddb")
-
-  (-> (build-connection-string)
-      refresh-db!))
-
-(deftask delete-qa
-  []
-  (System/setProperty "APP_ENV" "qa-ddb")
-
-  (-> (build-connection-string)
-      delete-db!))
-
-#_(deftask seed-prod
-  "Seeds the prod dynamo db"
-  []
-  (System/setProperty "APP_ENV" "prod-ddb")
-
-  (-> (build-connection-string)
-      create-db!
-      seed-db!))
+  (task-constructor "delete"
+                    #( -> (build-connection-string)
```

Comment:
The formatting here seems to be correct as far as I can tell, lmk if it still looks wrong to you.